### PR TITLE
Add test command to make

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,2 +1,6 @@
-flake: 
+test-quick:
+	env SKIP_MIGRATIONS=1 ./manage.py test -s --nologcapture
```
Comment (ar4s) [inline Makefile:2]: #159 introduce new command - `test_dnsaas` use this instead of `./manage.py`

File: Makefile [hunk 2]
```diff
@@ -1,2 +1,6 @@
...
+test:
+	./manage.py test -s --nologcapture
```
Comment (ar4s) [inline Makefile:4]: ditto

**Review by ar4s — CHANGES_REQUESTED** [review]

**Review by ar4s — APPROVED** [review]

**Review by mkurek — APPROVED** [review]

**Comment by quamilek** [comment]
@xliiv  It is ready to merge?

**Comment by ghost** [comment]
It depends on #159 and that one depends on making sure it will work.

**Pull request closed without merging**